rules/linux/credential_access_potential_linux_ssh_bruteforce.toml

```diff
@@ -17,6 +17,35 @@ index = ["auditbeat-*", "logs-system.auth-*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Potential SSH Brute Force Detected"
+note = """## Triage and analysis
+
```

Comment:
Should we add ### Investigating Potential SSH Brute Force Detected as a header here to remain consistent with the flow of other investigation guides?